testsupport/wait/member.go

```diff
@@ -312,6 +312,19 @@ func UntilSpaceRequestHasStatusTargetClusterURL(expected string) SpaceRequestWai
 	}
 }
 
+// UntilSpaceRequestHasNamespaceAccess returns a `SpaceRequestWaitCriterion` which checks that the given
+// SpaceRequest has `status.NamespaceAccess` set
+func UntilSpaceRequestHasNamespaceAccess() SpaceRequestWaitCriterion {
+	return SpaceRequestWaitCriterion{
+		Match: func(actual *toolchainv1alpha1.SpaceRequest) bool {
+			return len(actual.Status.NamespaceAccess) > 0
```

Comment:
could you also verify the expected number namespaces - there is only one. Plus also the name of the namespace? It shouldn't be hard to pass it to the `UntilSpaceRequestHasNamespaceAccess` function because at the point when you call this function you already know the sub-space name and the expected suffix `-env`

Reply:
As mentioned in the previous comment I did it but a bit differently. I'm comparing the number and name of namespaces with the ones from the `subspaces.Status.ProvisionedNamespaces`. 
In this way it will work also when we will have multiple namespaces. WDYT ? 

see: https://github.com/codeready-toolchain/toolchain-e2e/pull/706/commits/3f315e55a25920ece38c2351288e4faa2176ae66